internal/cli/organizations.go

```diff
@@ -461,33 +464,140 @@ auth0 orgs members ls <id>`,
-				})
+			members, err := cli.getOrgMembers(cmd.Context(), inputs.ID, inputs.Number)
+			if err != nil {
+				return err
+			}
+			sortMembers(members)
+			cli.renderer.MembersList(members)
+			return nil
+		},
+	}
+	return cmd
+}
+
+func rolesOrganizationCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "roles",
+		Short: "Manage roles of an organization",
+		Long:  "Manage roles of an organization.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listRolesOrganizationCmd(cli))
+	cmd.AddCommand(membersRolesOrganizationCmd(cli))
 
+	return cmd
+}
+
+func listRolesOrganizationCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID     string
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "List roles of an organization",
+		Long:    "List roles assigned to members of an organization.",
+		Example: `auth0 orgs roles list
+auth0 orgs roles ls <id>`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := organizationID.Pick(cmd, &inputs.ID, cli.organizationPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			members, err := cli.getOrgMembers(cmd.Context(), inputs.ID, inputs.Number)
 			if err != nil {
-				return fmt.Errorf("Unable to list members of an organization with Id '%s': %w", inputs.ID, err)
+				return err
 			}
-			var typedList []management.OrganizationMember
-			for _, item := range list {
-				typedList = append(typedList, item.(management.OrganizationMember))
+
+			roleMap := make(map[string]management.OrganizationMemberRole)
+			for _, member := range members {
+				userID := member.GetUserID()
+				roleList, err := cli.api.Organization.MemberRoles(inputs.ID, userID)
+				if err != nil {
+					return err
+				}
+				for _, role := range roleList.Roles {
+					roleID := role.GetID()
+					if _, exists := roleMap[roleID]; !exists {
+						roleMap[roleID] = role
+					}
+				}
+			}
+			// convert management.OrganizationMemberRole to management.Role
+			var roles []*management.Role
+			for _, role := range roleMap {
+				roles = append(roles, &management.Role{ID: role.ID, Name: role.Name, Description: role.Description})
 			}
-			cli.renderer.MembersList(typedList)
+			sort.Slice(roles, func(i, j int) bool {
+				return strings.ToLower(roles[i].GetName()) < strings.ToLower(roles[j].GetName())
+			})
+			cli.renderer.RoleList(roles)
 			return nil
 		},
 	}
+	return cmd
+}
+
+func membersRolesOrganizationCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		OrgID  string
+		RoleID string
+		Number int
+	}
+
+	cmd := &cobra.Command{
+		Use:   "members",
+		Args:  cobra.RangeArgs(1, 2),
+		Short: "List organization members for a role",
+		Long:  "List organization members that have a given role assigned to them.",
+		Example: `auth0 orgs roles members
+auth0 orgs roles members <org id> <role id>`,
```

Comment:
Please, have this command be `auth0 orgs roles members list <org-id> --role-id role`. See `--api-id` in https://auth0.github.io/auth0-cli/auth0_roles_permissions_add.html as an example.
`--role-id` (`-r`) should be a required flag.